main.py

```diff
@@ -1,12 +1,171 @@
 import requests
+import datetime
+from customers import Customers
+from videos import Videos
+from rentals import Rentals
 
-URL = "http://127.0.0.1:5000"
-BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
-def main():
-    print("WELCOME TO RETRO VIDEO STORE")
-    pass
 
+# URL = "http://127.0.0.1:5000"
+URL = "https://retro-video-store-api.herokuapp.com"
 
-if __name__ == "__main__":
-    main()
\ No newline at end of file
+
+def print_stars():
+    print("\n**************************\n")
+    
+
+
+def list_options():
+    options = {
+        "1": "Add a video", 
+        "2": "Edit a video",
+        "3": "Delete a video",
+        "4": "List all videos",
+        "5": "Get a video",
+        "6": "Add a customer",
+        "7": "Edit a customer",
+        "8": "delete a customer",
+        "9": "Get a customer",
+        "10": "Get all customers",
+        "11": "Check out a video",
+        "12": "Check in a video",
+        "00" : "Quit",   
+    }
```

Comment:
A future refactor could involve putting customer, video, and rental options in different menus.